build/commands/lib/util.js

```diff
@@ -722,6 +722,38 @@ const util = {
     return applyPatches()
   },
 
+  updateChromeVersion: () => {
```

Comment:
nit: It might be best to stop adding anything new to this weird util file, and add a new file at e.g. `build/commands/sync/updateChromeVersion.js` that simply exports a `function updateChromeVersion () { ...`

This function is only called in 1 place - sync.